stl/inc/yvals_core.h

```diff
@@ -1116,6 +1117,16 @@
 #define _HAS_STREAM_INSERTION_OPERATORS_DELETED_IN_CXX20 (_HAS_FEATURES_REMOVED_IN_CXX20)
 #endif // _HAS_STREAM_INSERTION_OPERATORS_DELETED_IN_CXX20
 
+
+#ifndef _HAS_FEATURES_REMOVED_IN_CXX23
+#define _HAS_FEATURES_REMOVED_IN_CXX23 (!_HAS_CXX23)

```

Comment:
Is that correct. The default would be that those Features are removed.

Semantically this appears to be flipped

Reply:
Same as https://github.com/microsoft/STL/blob/main/stl/inc/yvals_core.h#L1060-L1063

Yes. I think it's correct. It's name for users. If I as user define 
```
#define _HAS_FEATURES_REMOVED_IN_CXX23  1
```
then include some standard hearders 
and then all deleted features in c++23 are available for me.

Reply:
Yep, this is correct.

FYI: GitHub doesn't preview line-numbered links to files in `main`, because as `main` changes, the line numbers could be invalidated. However, if you press the `y` keyboard shortcut when viewing a file, the link will transform into a commit permalink, and then citing that in an issue/comment will be previewed. Here, `https://github.com/microsoft/STL/blob/1866b848f0175c3361a916680a4318e7f0cc5482/stl/inc/yvals_core.h#L1060-L1063` will be previewed as:

https://github.com/microsoft/STL/blob/1866b848f0175c3361a916680a4318e7f0cc5482/stl/inc/yvals_core.h#L1060-L1063